pkg/apis/migration.harvesterhci.io/v1beta1/virtualmachines.go

```diff
@@ -34,6 +41,11 @@ type VirtualMachineImportStatus struct {
 	DiskImportStatus  []DiskInfo         `json:"diskImportStatus,omitempty"`
 	ImportConditions  []common.Condition `json:"importConditions,omitempty"`
 	NewVirtualMachine string             `json:"newVirtualMachine,omitempty"`
+
+	// DefiniteVirtualMachineName is the sanitized and definite name of the
+	// target virtual machine that will be created in the Harvester cluster.
+	// The name is DNS1123 compliant.
+	DefiniteVirtualMachineName string `json:"definiteVirtualMachineName,omitempty"`
```

Comment:
this represents the imported VM. Maybe this should be called `ImportedVirtualMachineName` or `HarvesterVirtualMachineName`

Reply:
Renamed to `ImportedVirtualMachineName`.